### PR TITLE
feat: add key and subkey to view/set token area

### PR DESCRIPTION
**Pull request by Marianarodrigues7**
Add key and subkey when viewing the value or setting it.

## screenshots
image

### CODE REVIEW

**Review by JPedroBorges — APPROVED** [review]
What a great decision on using `fw-lighter` 💣
Looks super cool.
Let me know what you think @Marianarodrigues7

File: src/components.js [hunk 1]
```diff
@@ -122,6 +125,38 @@ function appComponents() {
     return textArea;
   }
 
+  function newKeyDetails({ id, key, subKey }) {
```
Comment (Marianarodrigues7) [inline src/components.js:128]: Probably not for this release, but I don't agree on building a key details in components. Key details is a specific component of key list. What I believe we should do in components is to create a `newBadge` and probably a new `newLabelWithBadge`. The rest of the keyList specific logic should be built in items and pass down to the footer body. WDYT @JPedroBorges ?

Comment (Marianarodrigues7) [reply src/components.js:128]: Also, just to add to the previous comment, things like key or subkey should decoupled from components. If in the future we add another property related to our data, to be shown on this "header", we should not need to change components.

File: src/items.js [hunk 2]
```diff
@@ -78,7 +83,10 @@ const appCreator = (components) => {
         },
       });
       return {
-        body: tokenArea,
+        body: {
+          tokenArea,
+          keyDetails,
+        },
```
Comment (Marianarodrigues7) [inline src/items.js:89]: Adding to the comment in components, I believe the body should already be built and pass down to components in its final form.

File: src/page.js [hunk 3]
```diff
@@ -201,21 +201,25 @@ function appPage(creator, {storeSave, storeDelete, storeSet, refreshValues}) {
...
   }
 
-  function onViewValue(itemId, element) {
+  function onViewValue(id, element) {
     analytics.fireEvent("view_value");
-    const textArea = document.getElementById(`token-${itemId}`);
+    const textArea = document.getElementById(`token-${id}`);
     textArea.disabled = true;
+    const keyDetails = document.getElementById(`key-${id}`);
+    keyDetails.classList.add("display-none");
```
Comment (Marianarodrigues7) [inline src/page.js:209]: We want to show these values when clicking `view value`
```suggestion
    keyDetails.classList.remove("display-none");
```

File: src/components.js (hunk 1 above)
Comment (Marianarodrigues7) [reply src/components.js]: Fixed ✅

File: src/page.js (hunk 3 above)
Comment (Marianarodrigues7) [reply src/page.js]: Removed ✅

File: src/items.js (hunk 2 above)
Comment (Marianarodrigues7) [reply src/items.js]: Fixed ✅

**Pull request merged**